internal-enrichment/virustotal/src/virustotal/virustotal.py

```diff
@@ -206,7 +693,283 @@ def _process_message(self, data):
             raise ValueError(
                 "Do not send any data, TLP of the observable is greater than MAX TLP"
             )
-        return self._process_file(observable)
+
+        if (
+            observable["entity_type"] == "StixFile"
+            or observable["entity_type"] == "Artifact"
+        ):
+            return self._process_file(observable)
+        elif observable["entity_type"] == "IPv4-Addr":
+            return self._process_ip(observable)
+        elif observable["entity_type"] == "Domain-Name":
+            return self._process_domain(observable)
+        elif observable["entity_type"] == "Url":
+            return self._process_url(observable)
+        else:
+            raise ValueError(
+                f'{observable["entity_type"]} is not a supported entity type.'
+            )
+
+    def _country_code_to_country_name(self, code):
```

Comment:
Is it needed to create new countries or could the connector expect the user to have the OpenCTI country dataset imported? https://luatix.notion.site/Import-Export-7dc143dfbb6147b0881080487ed9db33#ba79dc9199eb47119117a1a6669b6316 This would avoid creating new countries.